doc/configuration-flow.md

```diff
@@ -0,0 +1,54 @@
+# Overview of the configuration flow in Istio Manager
+
+Istio configuration consists of configuration objects. Each object has kind
+(for example, a routing rule), name and namespace uniquely identifying the
+object.  Istio Manager provides a REST-based interface for manipulating objects
```

Comment:
A --debug option that displays the equivalent HTTP/curl request that the command is doing might be helpful for understanding